packages/backend-core/package.json

```diff
@@ -45,7 +45,7 @@
     "passport-oauth2-refresh": "^2.1.0",
     "pino": "8.11.0",
     "pino-http": "8.3.3",
-    "posthog-node": "1.3.0",
+    "posthog-node": "4.0.1",
```

Comment:
This was required in order to fetch all feature flags at once. `posthog.getAllFeatureFlagsAndPayloads` didn't exist in 1.3.0.